src/Microsoft.ApplicationInsights/Properties/AssemblyInfo.cs

```diff
@@ -10,6 +10,8 @@
 [assembly: InternalsVisibleTo("Microsoft.ApplicationInsights.netcoreapp11.Tests" + AssemblyInfo.PublicKey)]
 [assembly: InternalsVisibleTo("Microsoft.ApplicationInsights.netcoreapp20.Tests" + AssemblyInfo.PublicKey)]
 
+
+[assembly: InternalsVisibleTo("Microsoft.AI.ServerTelemetryChannel" + AssemblyInfo.PublicKey)]
```

Comment:
We typically try to avoid InternalsVisibleTo if possible, let's continue our offline discussion to see if it's still necessary.